source/faq.rst

```diff
@@ -216,5 +216,44 @@ You can change the minimum resolution for metrics by the following way:
    when |pmm-server| and |pmm-client| are on different networks,
    or when :ref:`pmm.amazon-rds`.
 
+.. _alertmanager:
+
+How to set up Alerting in PMM?
+================================================================================
+
+You can make PMM Server firing alerts when your monitored service reaches some thresholds in two ways:
+
+* using `Grafana Alerting feature <https://grafana.com/docs/grafana/latest/alerting/rules/>`_,
+* using external `Alertmanager <https://github.com/prometheus/alertmanager>`_ (a
+  high-performance solution developed by the Prometheus project to handle alerts sent
+  by Prometheus).
+
+Both options can be considered advanced features and require knowledge of
+third-party documentation.
+
+Either with Grafana Alerting or with Alertmanager you need to configure some
+alerting rule to define conditions under which the alert should be fired out,
+and some channel to which the alert is to be sent (e.g. e-mail).
+
+Grafana Alerts can be preferred because of being integrated into the PMM Server,
```

Comment:
how about 'Grafana Alerts are already integrated into PMM Server and may be simpler to get set up,'